test/reazon-test-interface.el

```diff
@@ -47,6 +47,19 @@
   (reazon--should-equal '(olive _0 oil)
     (reazon-run* x (reazon-disj (reazon-conj (reazon-== 'virgin x) #'reazon-!U) (reazon-disj (reazon-== 'olive x) (reazon-disj #'reazon-!S (reazon-== 'oil x)))))))
 
+(reazon-defrel reazon-test--alwayso ()
+  "Infinite successful goals."
+  (reazon-disj #'reazon-!S (reazon-test--alwayso)))
+
+(reazon-defrel reazon-test--nevero ()
+  "Infinite unsuccessful goals."
+  (reazon-disj #'reazon-!U (reazon-test--nevero)))
+
+(ert-deftest reazon-test-interface-run-with-config ()
+  (reazon--should-equal '()
+    (reazon-run-with-config nil q '(:timeout 0.01) (reazon-test--nevero)))
+  (should (consp (reazon-run-with-config nil q '(:timeout 0.01) (reazon-test--alwayso)))))
```

Comment:
```suggestion
(ert-deftest reazon-test-interface-run-with-config ()
  (reazon--should-equal '()
    (reazon-run-with-config nil q '(:timeout 0.01)
      (reazon-test--nevero)))
  (should
   (consp
    (reazon-run-with-config nil q '(:timeout 0.01)
      (reazon-test--alwayso)))))
```

Reply:
Can we do any better than `consp`?

Reply:
Switched this to a `length >3` check, abritrarily chose 3